tests/test_basics.py

```diff
@@ -494,6 +498,73 @@ def _fake_import(name, *args):
         qt_api.set_qt_api(api=None)
 
 
+@pytest.mark.parametrize(
+    "option_api, backend",
+    [
+        ("pyqt5", "PyQt5"),
+        ("pyqt6", "PyQt6"),
+        ("pyside2", "PySide2"),
+        ("pyside6", "PySide6"),
+    ],
+)
+def test_already_loaded_backend(monkeypatch, option_api, backend):
+
+    import builtins
+
+    class Mock:
+        pass
+
+    qtcore = Mock()
+    for method_name in (
+        "qInstallMessageHandler",
+        "qDebug",
+        "qWarning",
+        "qCritical",
+        "qFatal",
+    ):
+        setattr(qtcore, method_name, lambda *_: None)
+
+    if backend in ("PyQt5", "PyQt6"):
+        pyqt_version = 0x050B00 if backend == "PyQt5" else 0x060000
+        setattr(qtcore, "PYQT_VERSION", pyqt_version + 1)
```

Comment:
Any reason why you are using `setattr` instead of just setting the attribute, like:

```suggestion
        qtcore.PYQT_VERSION = pyqt_version + 1
```